src/btree/bt_sync_obsolete.c

```diff
@@ -477,8 +479,16 @@ __checkpoint_cleanup_eligibility(WT_SESSION_IMPL *session, const char *uri, cons
         if (ret == 0)
             newest_stop_durable_ts = WT_MAX(newest_stop_durable_ts, (wt_timestamp_t)value.val);
         WT_RET_NOTFOUND_OK(ret);
+        ret = __wt_config_subgets(session, &cval, "addr", &value);
+        if (ret == 0)
+            addr_size = value.len;
+        WT_RET_NOTFOUND_OK(ret);
     }
 
+    /* Ignore the tables that are empty or newly created. */
+    if (addr_size == 0)
```

Comment:
Can we add this condition to the one below and explain why we need it to be greater than 0?

Reply:
Moved it into the below check code.